src/engine/QueryExecutionTree.cpp

```diff
@@ -118,7 +118,7 @@ QueryExecutionTree::setPrefilterGetUpdatedQueryExecutionTree(
     return std::nullopt;
   } else {
     return rootOperation_->setPrefilterGetUpdatedQueryExecutionTree(
-        std::move(prefilterPairs));
+        prefilterPairs);
```

Comment:
This in fact is not so good, using a mutable reference here.
If you have some operation with two children, then you get really surprising results, the old code with
the value + std::move was okay, sorry for the confusion...